scripts/azure_deployment/arm_aci.py

```diff
@@ -489,13 +479,16 @@ def check_aci_deployment(
 
             while current_time < end_time:
                 try:
+                    print(
```

Comment:
For the sake of documentation, further improvement could be to use az container exec to setup keys in which case this check of ssh connection won't be necessary